aml/org.testeditor.aml.model/pom.xml

```diff
@@ -46,7 +46,7 @@
 						</execution>
 					</executions>
 					<configuration>
-						<failOnValidationError>true</failOnValidationError>
+						<classPathLookupFilter>.*(ecore|genmodel|xcore|xtext).*</classPathLookupFilter>
```

Comment:
Sorry wrong pom. If we use a generic swt fixture, we will need it for the uatest project.